src/pages/Payment.vue

```diff
@@ -90,7 +86,7 @@ export default {
     },
     methods: {
         showConfirmation: function() {
-            this.setMessage("Thanks, your payment request has been sent.", constants.MESSAGE_SUCCESS);
+            this.setMessage("Thanks, your payment request has been sent. Please take your vouchers to your market representative for them to be sent off.", constants.MESSAGE_SUCCESS);
```

Comment:
I guess depends on if we will need to show instructions in the message - and by request - (help or payment instructions) or if it is only in this place.